org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/core/manipulation/OrganizeImportsOperation.java

```diff
@@ -411,6 +411,13 @@ private TypeNameMatch[] processTypeInfo(List<TypeNameMatch> typeRefsFound) {
 			if (nFound == 0) {
 				// nothing found
 				return null;
+			//} else if (fDoPreserveDemandImports && fOldDemandImports.contains(typeRefsFound.get(0).getTypeContainerName())) {
```

Comment:
Should not add a commented out line.

Reply:
I have eliminated commented out code line.